docker/runtime/backup-job.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+execute_backup_job() {
+	BACKUP_ARG_PATH="${1:-}"
+	BACKUP_ARG_CONFIG="${BACKUP_ARG_CONFIG:-}"
+	BACKUP_ARG_EXCLUDE_REPOS="${BACKUP_ARG_EXCLUDE_REPOS:-}"
+	cd "/app/gogs" || exit 1
+
+	BACKUP_ARGS="--target=${BACKUP_ARG_PATH}"
+
+	if [ -n "${BACKUP_ARG_CONFIG}" ]; then
+		BACKUP_ARGS="${BACKUP_ARGS} --config=${BACKUP_ARG_CONFIG}"
+	fi
+
+	if [ -n "${BACKUP_ARG_EXCLUDE_REPOS}" ]; then
+		BACKUP_ARGS="${BACKUP_ARGS} --exclude-repos=${BACKUP_ARG_EXCLUDE_REPOS}"
+	fi
+
+	./gogs backup "${BACKUP_ARGS}" || echo "Error: Backup job returned non-successful code." && exit 1
```

Comment:
qq: If an error occurred, where can user find the output for debugging?

Reply:
It's possible to see it in docker log stream with `docker logs gogs` or inside Portainer interface (if present).